src/catalog/catalog.cpp

```diff
@@ -228,14 +228,14 @@ table_id_t CatalogContent::addRelTableSchema(string tableName, RelMultiplicity r
     auto propertyNameDataType = PropertyNameDataType(INTERNAL_ID_SUFFIX, INT64);
     structuredProperties.push_back(
         Property::constructRelProperty(propertyNameDataType, propertyID++, tableID));
-    for (auto& propertyDefinition : structuredPropertyDefinitions) {
+    for (auto& propertyDefinition : propertyDefinitions) {
         structuredProperties.push_back(
```

Comment:
Since we only have structuredProperties now, i think we can rename `structuredProperties`->`properties`

Reply:
Done.